draft-pauly-oblivious-doh.md

```diff
@@ -240,14 +240,58 @@ struct {
    uint16 kdf_id;
    uint16 aead_id;
    opaque public_key<1..2^16-1>;
-} ObliviousDNSKey;
+} ObliviousDoHConfigContents;
+
+struct {
+   uint16 version;
+   uint16 length;
+   select (ObliviousDoHConfig.version) {
```

Comment:
`select` is not an [XDR type](https://tools.ietf.org/html/rfc4506) so I don't think an IETF draft should use it. We could use an `enum` instead to represent the same.

Reply:
For better or worse, this is standard in IETF specifications. See, for example, RFC8446.